test/WebJobs.Script.Tests/Metrics/LinuxContainerLegionMetricsPublisherTests.cs

```diff
@@ -45,7 +45,7 @@ public LinuxContainerLegionMetricsPublisherTests()
             _environment = new TestEnvironment();
             _logger = new TestLogger<LinuxContainerLegionMetricsPublisher>();
             _testMetricsLogger = new TestMetricsLogger();
-            _scriptHostManager = new TestScriptHostManager(_testMetricsLogger);
+            _serviceProvider = new TestScriptHostManager(_testMetricsLogger);
 
```

Comment:
I'd like to see a test in here somewhere that would have failed before your change.

Reply:
Looks like he's added one now?

Reply:
Yes. Added a new test. https://github.com/Azure/azure-functions-host/pull/10991#discussion_r2039722311